ftplugin/pkl.vim

```diff
@@ -15,6 +15,7 @@
 " Fold using tree-sitter
 setlocal foldmethod=expr
 setlocal foldexpr=nvim_treesitter#foldexpr()
+set foldlevelstart=99
```

Comment:
This change isn't working for me. When opening a Pkl file, everything still starts out as folded.

Instead, I'm thinking it's probably better to just remove `setlocal foldmethod=expr`; it's unnecessarily opinionated and has an unwanted side-effect if set.

Reply:
That makes sense. In the future, this could also be a config option, but right now I just wanted to fix the folding issue easily.